h2o-py/tests/pyunit_utils/utils_for_glm_tests.py

```diff
@@ -0,0 +1,150 @@
+import h2o
+from h2o.estimators import H2OGeneralizedLinearEstimator as glm
+from h2o.exceptions import H2OValueError
+from h2o.grid.grid_search import H2OGridSearch
+
```

Comment:
```suggestion


```
Ideally, I would try to stick to PEP-8 and [keep 2 blank lines between top-level functions](https://peps.python.org/pep-0008/#blank-lines).